k8s/src/main/scala/io/buoyant/k8s/resources.scala

```diff
@@ -28,7 +30,9 @@ trait Resource extends Closable {
  * @tparam O The parent type for all [Object]s served by this version.
  */
 private[k8s] trait Version[O <: KubeObject] extends Resource {
-  val path = s"/$group/$version"
+
+  override val path = s"/$group/$version"
+  override val watchPath = s"/$group/$version/watch"
```

Comment:
Yeah, it was like that for a while, but then I changed these to `lazy val`s based on the observation that we probably wouldn't have to allocate both strings. @adleong quashed that & I never changed it back...